packages/client/src/comp/states/Base.test.tsx

```diff
@@ -217,3 +219,32 @@ test('Signup Flow allows for separate mailing address', async () => {
   await wait(() => expect(register).toHaveBeenCalledTimes(1))
   await compareResults(register, true)
 })
+
+test("Sends fax to Twillio successfully", async () => {
```

Comment:
You need to test a fax state (Wisconsin is not a fax state).  The list of fax states are listed here: http://localhost:8080/dashboard

You should test one (e.g. Nevada) and make sure `sendFaxes` is called with an e164 number.

Reply:
sendFaxes is a function in the backend side therefore I was not able to mock it, I could not find a way to get inside the register function (where the sendFaxes is called) and see if sendFaxes was called, apart from testing if the e164 number was formatted the right way.

Reply:
You're right.  This should be a test on the backed (the `register` function).  We don't need to test `Base.test.tsx` -- sorry, wasn't paying attention to the file.